embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    corpus_df = pd.DataFrame()
</pre><p>Stylistic thing: Could you load all of the fields when creating the dataframe? It can be easier to read</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>done</p>